lib/Conversation/index.js

```diff
@@ -29,41 +34,55 @@ class Conversation extends Component {
     this.props.actions.resolveConversation(this.props.id);
   }
 
-  handleConvoClick() {
+  activateConversation() {
+    this.props.actions.activateConversation(this.props.id);
     this.setState({
       isActive: true,
     });
   }
 
+  deactivateConversation() {
+    this.props.actions.deactivateConversation(this.props.id);
+    this.setState({
+      isActive: false,
+    });
+  }
+
   render() {
     const conversationClasses = (this.state.isActive) ? 'conversation--is-active' : '';
```

Comment:
Just a reminder that we have the `classnames` (cx) package available to help us handle conditional class names! It's okay for these because it's either on or off 😉